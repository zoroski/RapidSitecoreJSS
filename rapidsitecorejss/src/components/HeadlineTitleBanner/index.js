import React from 'react';
import { Text } from '@sitecore-jss/sitecore-jss-react';

const HeadlineTitleBanner = (props) => (
  <div>
    <p>HeadlineTitleBanner Component</p>
    <Text field={props.fields.heading} />
  </div>
);

export default HeadlineTitleBanner;
