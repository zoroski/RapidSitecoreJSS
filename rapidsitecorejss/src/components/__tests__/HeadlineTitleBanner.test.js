import React from 'react';
import HeadlineTitleBanner from '../HeadlineTitleBanner/index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
      const tree = renderer
        .create( < HeadlineTitleBanner fields = {
            {
              heading: {
                value: 'Heading',
              },
            }
          }
          />)
          .toJSON(); expect(tree).toMatchSnapshot();
        });