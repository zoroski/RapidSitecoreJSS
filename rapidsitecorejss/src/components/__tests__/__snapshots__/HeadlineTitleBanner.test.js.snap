// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2
    className="display-4"
  >
    Heading
  </h2>
</div>
`;
